ospd/ospd.py

```diff
@@ -1423,6 +1423,9 @@ def handle_command(self, command: str) -> str:
         if not command and tree.tag != "authenticate":
             raise OspdCommandError('Bogus command name')
 
+        if tree.tag == "get_vts":
+            return command.handle_xml(tree, stream)
```

Comment:
Personally i wouldn't pass the stream to the commands. The commands should yield data

```python
for data in command.handle_xml(tree):
    stream.write(data)